perfkitbenchmarker/linux_benchmarks/hpcc_benchmark.py

```diff
@@ -106,7 +106,7 @@ def CreateHpccinf(vm, benchmark_spec):
   if FLAGS.memory_size_mb:
```

Comment:
Can you invoke DropCaches before grabbing the free memsize?
https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/6e0e86d073aa9567b704a8b452970d5826c6fbfb/perfkitbenchmarker/linux_virtual_machine.py#L493